_dropin-templates/dropin-containers.mdx

```diff
@@ -1,82 +1,145 @@
 ---
-title: DROPIN_NAME containers and configuration
-description: Learn about the containers and configurations for the DROPIN_NAME drop-in.
+title: ContainerName container
+description: Learn about the ContainerName container and configurations for the DROPIN_NAME drop-in component.
 ---
 
-import Diagram from '@components/Diagram.astro';
-import Callouts from '@components/Callouts.astro';
 import OptionsTable from '@components/OptionsTable.astro';
-import Aside from '@components/Aside.astro';
-import { Steps } from '@astrojs/starlight/components';
 
-This section describes the containers and configuration options for the DROPIN_NAME.
+The ContainerName container [describe what the container does]. You can customize it with slots, run separate instances, and it takes care of loading, errors, and real-time updates.
 
-## Big Picture
+## Import
 
-{/* Replace `placeholder.webp` with a diagram identifying all its containers. */}
-
-<Diagram caption="DROPIN_NAME containers">
-  ![DROPIN_NAME containers](@images/placeholder.webp)
-</Diagram>
-
-{/* Use `Callouts` control to name the containers in the drop-in. */}
-
-<Callouts>
-
-{/* Use a numbered list to identify the slot names. */}
-
-1. **CONTAINER_NAME_1**: Provides the DESCRIBE_PURPOSE.
-1. **CONTAINER_NAME_2**: Provides the DESCRIBE_PURPOSE.
-   {/* Example 1: **ProductDetails**: Provides the layout and configuration options for displaying the details of a product. */}
-
-</Callouts>
-
-## CONTAINER_NAME_1 configurations
+```javascript
+import ContainerName from '@dropins/storefront-DROPIN_PACKAGE/containers/ContainerName.js';
+```
 
-The CONTAINER_NAME_1 container provides the following configuration options:
+## Configurations
 
-{/* Use the OptionsTable to list and describe each option. */}
+The `ContainerName` container provides the following configuration options:
 
 <OptionsTable
   compact
   options={[
     ['Option', 'Type', 'Req?', 'Description'],
-    ['NAME', 'TYPE', 'YES or NO', 'DESCRIPTION'],
+    ['option1', 'string', 'No', 'Description of option1. Default: value.'],
+    ['option2', 'function', 'No', 'Description of option2. Receives parameter.'],
+    ['slots', 'object', 'No', 'Custom slot renderers for customization.'],
   ]}
 />
 
-{/* EXAMPLE: */}
-{/* <OptionsTable compact options={[ */}
-{/* ['Option', 'Type', 'Req?', 'Description'], */}
-{/* ['hideSku', 'boolean', 'No', 'Optional boolean to hide the SKU display.'], */}
-{/* ]}/> */}
+## Basic Usage
+
+```javascript
+// Basic container usage
+await provider.render(ContainerName, {
+  option1: 'value',
+  option2: (param) => `/path/${param.property}`
+})($container);
+
+// With scope for isolated instance
```

Comment:
* if supported. Not all drop-ins have added (or will add) support to scoping.

Reply:
@fnhipster I'll just remove the code block.